src/Docker.DotNet/Endpoints/IContainerOperations.cs

```diff
@@ -73,7 +73,7 @@ public interface IContainerOperations
         /// <param name="cancellationToken">When triggered, the operation will stop at the next available time, if possible.</param>
         /// <returns>A <see cref="Task{TResult}"/> that resolves to a <see cref="Stream"/>, which provides the log information.</returns>
         /// <remarks>This method works only for containers with the <c>json-file</c> or <c>journald</c> logging driver.
-        /// <br/>The corresponding commands in the Docker CLI are <c>docker logs</c> and <c>docker container logs</c>.</remarks>
+        /// <br/>The corresponding command in the Docker CLI is <c>docker logs</c>.</remarks>
```

Comment:
```suggestion
        /// <br/>The corresponding commands in the Docker CLI are <c>docker logs</c> and <c>docker container logs</c>.</remarks>
```

Both are valid Docker CLI commands.

Reply:
fixed